aladdin.sh

```diff
@@ -263,28 +247,26 @@ function handle_ostypes() {
     esac
 }
 
-function prepare_docker_subcommands() {
+function enter_docker_container() {
+
     # if this is not production or staging, we are mounting kubernetes folder so that
     # config maps and other settings can be customized by developers
+    local -a mounts
     if "$DEV"; then
-        DEV_CMD="-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin"
-        DEV_CMD="$DEV_CMD -v /:/minikube_root" # mount the whole minikube system
-        DEV_CMD="$DEV_CMD --workdir /minikube_root$(pathnorm "$PWD")"
+        mounts=("${mounts[@]}" "-v $(pathnorm "$ALADDIN_DIR"):/root/aladdin")
     fi
-
-    # If on minikube mount minikube credentials
     if "$IS_LOCAL"; then
-        MINIKUBE_CMD="-v $(pathnorm ~/.minikube):/root/.minikube"
-        MINIKUBE_CMD="$MINIKUBE_CMD -v /:/minikube_root" # mount the whole minikube system
-        MINIKUBE_CMD="$MINIKUBE_CMD --workdir /minikube_root$(pathnorm "$PWD")"
+        mounts=("${mounts[@]}" "-v $(pathnorm ~/.minikube):/root/.minikube")
+    fi
+    if "$IS_LOCAL" || "$DEV"; then
+        mounts=("${mounts[@]}" "-v /:/aladdin_root")
+        mounts=("${mounts[@]}" "--workdir /aladdin_root$(pathnorm "$PWD")")
```

Comment:
Same here.  If you wouldn't put quotes around it on the command line, you shouldn't here.